fyle_accounting_mappings/models.py

```diff
@@ -332,10 +335,15 @@ def bulk_upsert_mapping_setting(settings: List[Dict], workspace_id: int):
 
         with transaction.atomic():
             for setting in settings:
-                mapping_setting, _ = MappingSetting.objects.get_or_create(
+                mapping_setting, _ = MappingSetting.objects.update_or_create(
                     source_field=setting['source_field'],
                     workspace_id=workspace_id,
-                    destination_field=setting['destination_field']
+                    destination_field=setting['destination_field'],
+                    expense_field_id=setting['expense_field_id'],
+                    defaults={
+                        'import_to_fyle': setting['import_to_fyle'],
```

Comment:
Can we do `setting['import_to_fyle'] if 'import_to_fyle' in setting` else False.
Same for `is_custom` too. I would not like to send these keys from configurations page mappings post.
Will defer to Shwetabh for this anyways.

Reply:
Yes, have this doubt already, I would prefer doing this check here itself ☮️